src/main/java/com/github/princesslana/eriscasper/data/util/Nullable.java

```diff
@@ -27,6 +27,10 @@ private Nullable(Optional<T> value) {
     return of(value.map(f));
   }
 
+  public T orElse(T other) {
```

Comment:
So the reason it's there was because I needed it in the QueryStringBuilder (if you look back into the commits) and I just never removed it. We can remove it if you'd like, I have no preference to it, but with some people, it may be useful as an API option considering we're not just using it ourselves but making our users use it as well.